apps/frontend/src/pages/analytics/analytics.tsx

```diff
@@ -35,16 +41,25 @@ const Analytics = (): JSX.Element => {
 		void dispatch(activityLogActions.loadAllProjects());
 	}, [dispatch]);
 
-	const { control, handleSubmit, isDirty } = useAppForm({
+	const { control, errors, handleSubmit, isDirty } = useAppForm({
 		defaultValues: {
 			dateRange: [
 				subtractDays(todayDate, ANALYTICS_DATE_MAX_RANGE),
 				todayDate,
 			] as [Date, Date],
 			project: null,
+			search,
 		},
+		mode: "onChange",
```

Comment:
We have not any validation for search, so there is no need for this right now. Right?
```suggestion
```